integration/simple-cache-client.test.ts

```diff
@@ -19,24 +25,31 @@ const deleteCacheIfExists = async (
   momento: SimpleCacheClient,
   cacheName: string
 ) => {
-  try {
-    await momento.deleteCache(cacheName);
-  } catch (e) {
-    if (!(e instanceof NotFoundError)) {
-      throw e;
+  const deleteResponse = await momento.deleteCache(cacheName);
+  if (deleteResponse instanceof DeleteCache.Error) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
why does it think that `deleteResponse.errorCode()` is potentially unsafe?

Reply:
I honestly don't know . . . let me know if you'd like to look into it with me.

Reply:
I would like to look into this, I can find some time today.  Not necessarily a blocker for the PR though.